tests/test_data.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+test_data
+Test for Dataset class 
+@author: jdiedrichsen
+"""
+
+import unittest 
+import pyrsa.data as rsd
+import numpy as np 
+
+class TestSimulation(unittest.TestCase): 
```

Comment:
See my comments in #17, should this be named `TestSimulation`?

Reply:
Will work on this next week, and add more test cases. Thanks for the note!

Reply:
fixed and pushed. As well as including test cases for initialization and four methods, all past unit tests.